examples/api/test/widgets/scroll_view/custom_scroll_view.1_test.dart

```diff
@@ -0,0 +1,244 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/gestures.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_api_samples/widgets/scroll_view/custom_scroll_view.1.dart' as example;
+import 'package:flutter_test/flutter_test.dart';
+
+void main() {
+  testWidgets('Widget tree is visible', (WidgetTester tester) async {
+    await tester.pumpWidget(const example.CustomScrollViewExampleApp());
+
+    // The crucial Widgets are:
+    // - Scaffold
+    //    - Appbar
+    //      - IconButton
+    //    - CustomScrollView
+    //       - SliverList (top, initially not existing)
+    //       - SliverList (bottom, with one element)
+
+    expect(
+      find.byType(Scaffold),
+      findsOne,
+      reason: 'Expected to have a Scaffold in the App',
+    );
+
+    expect(
+      find.descendant(
+        of: find.byType(Scaffold),
+        matching: find.byType(AppBar),
+      ),
+      findsOne,
+      reason: 'Expected to have an Appbar in the Scaffold',
+    );
+
+    expect(
+      find.descendant(
+        of: find.byType(AppBar),
+        matching: find.byType(IconButton),
+      ),
+      findsOne,
+      reason: 'Expected an IconButton in the AppBar',
+    );
+
+    expect(
+      find.descendant(
+        of: find.byType(Scaffold),
+        matching: find.byType(CustomScrollView),
+      ),
+      findsOne,
+      reason: 'Expected a CustomScrollView in the Scaffold',
+    );
+
+    // Initially, there is only one SliverList
+    expect(
+      find.descendant(
+        of: find.byType(CustomScrollView),
+        matching: find.byType(SliverList),
+      ),
+      findsOne,
+      reason: 'Expected one SliverList in the CustomScrollView',
+    );
+
+    //CustomScrollView contains one element 'Item: 0'
+    expect(
+      find.descendant(
+        of: find.byType(CustomScrollView),
+        matching: find.text('Item: 0'),
+      ),
+      findsOne,
+      reason: 'Expected element with text "Item: 0" in the CustomScrollView',
+    );
+  });
+  testWidgets('IconButton click extends existing SliverList',
+      (WidgetTester tester) async {
```

Comment:
Having both of the testWidgets arguments start on the same line, with a line break in between calls, would be ideal for consistency with other tests in the framework.

```suggestion

  testWidgets('IconButton click extends existing SliverList', (WidgetTester tester) async {
```